### PR TITLE
TecplotOutput: Additional options

### PR DESCRIPTION
**Pull request by ThieJan**
New keywords for tecplot output to
        - add output id to filename: enable multiple output on same domain
	- write node numbers to tecplot file

### CODE REVIEW

File: FEM/Output.cpp [hunk 1]
```diff
@@ -561,6 +567,24 @@ ios::pos_type COutput::Read(std::ifstream& in_str,
...
+            continue;
+        }
+        // For tecplot node-nr output JM
+        if (line_string.find("$NODE_INDEX_OUTPUT") != string::npos)
+        {
+            _node_output = true;
+            continue;  // JM
+        }
+        // For tecplot quadratic node output JM
+        if (line_string.find("$QUAD_NODE") != string::npos)
```
Comment (norihiro-w) [inline FEM/Output.cpp]: minor comment: To me, the name "QUAD_NODE" is not clear about what it does. I would recommend QUAD_NODE_OUTPUT.

File: FEM/Output.cpp [hunk 2]
```diff
@@ -561,6 +567,24 @@ ios::pos_type COutput::Read(std::ifstream& in_str,
...
+        // for tecplot: Number output files for multiple output on same domain
+        if (line_string.find("$NUMBER_TEC_OUTPUT") != string::npos)
+        {
+            _number_outputs = true;
+            continue;
+        }
+        // For tecplot node-nr output JM
+        if (line_string.find("$NODE_INDEX_OUTPUT") != string::npos)
+        {
+            _node_output = true;
```
Comment (norihiro-w) [inline FEM/Output.cpp]: the variable name might be misleading. how about renaming it to "_node_id_output"?

File: FEM/Output.cpp (hunk 1 above)
Comment (ThieJan) [reply FEM/Output.cpp]: Since it is a subkeyword of #OUTPUT, I don't know if this is necesary. However, I will rename it.

**Comment by norihiro-w** [comment]
👍

**Comment by ThieJan** [comment]
I do not understand why this tests fails. On my local maschine it fails using gcc 8.3 but works with gcc 7.4

**Comment by wenqing** [comment]
@ThieJan The failed test has nothing to do with this PR.

**Pull request merged**